src/consts/chains.ts

```diff
@@ -1,4 +1,5 @@
-import { ChainMap, ChainMetadata, ExplorerFamily } from '@hyperlane-xyz/sdk';
+import { ChainMap, ChainMetadata, ChainTechnicalStack, ExplorerFamily } from '@hyperlane-xyz/sdk';
+import { etherToken } from '@hyperlane-xyz/sdk/dist/consts/chainMetadata';
```

Comment:
How about using another chain's nativeToken (e.g. `ethereum.nativeToken`) instead of reaching into an unexported value like this? Imports into packages should go through the front door, with occasional exceptions for types or code that won't get bundled for production

Reply:
sgtm!